docs/data_types/tensor/tensor.md

```diff
@@ -0,0 +1,118 @@
+# Tensor
+
+
+
+DocArray support several tensor type that can be used inside BaseDoc. 
+
+The main ones are : 
+- [`NdArray`][docarray.typing.tensor.NdArray] for numpy tensor
+- [`TorchTensor`][docarray.typing.tensor.TorchTensor] for Pytorch tensor
+- [`TensorFlowTensor`][docarray.typing.tensor.TensorFlowTensor] for tensorflow tensor
+
+The three of them inherit from their respective framework tensor type. This means that they can be used natively inside their framework.
```

Comment:
This is not true for tensorflow. For the other ones i would stress this point more.

Reply:
fair